examples/DropletCollision/README.md

```diff
@@ -0,0 +1,18 @@
+# Droplet Collision
+
+## Description
+Two spherical liquid droplets (LJTS) are set in vapor next to each other. A macroscopic velocity was assigned to them so that they will collide eventually.
+
+## Setup
+The provided script will setup the initial checkpoint including the two droplets. To make the script run, make sure that the ls1 python toolbox [ppls1](https://github.com/ls1mardyn/ls1-mardyn/tree/master/tools/ppls1/ppls1) works properly. Change the paths in the python script according to the present system.
+
+## Steps
+The script will execute three steps, represented in the specified working folder as `s1_Bulk-Liquid`, `s2_Droplet-Equi`, and `s3_Droplet_Collision`.
```

Comment:
After reading the rest of the README this initial sentence seems like lie :P

```suggestion
The script will execute the two initialization steps of the simulation, represented in the specified working folder as `s1_Bulk-Liquid` and `s2_Droplet-Equi`. It will also prepare the main simulation `s3_Droplet_Collision` for execution.
```